app/models/collection.js

```diff
@@ -46,6 +45,7 @@ class Collection {
     }
 
     save() {
+        this.bullets = this.bullets.map(bullet => bullet.id);  //beforeSave, converts bullet instances to ids
```

Comment:
There needs to be a conditional check to see if the bullets are just ids or if they're bullet instances